### PR TITLE
Declaration files for JMuxer

### PR DESCRIPTION
**Pull request by samirkumardas**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

Select one of these and delete the others:

If adding a new definition:
- [x] The package does not already provide its own types, or cannot have its `.d.ts` files generated via `--declaration`
- [x] If this is for an npm package, match the name. If not, do not conflict with the name of an npm package.
- [x] Create it with `dts-gen --dt`, not by basing it on an existing project.
- [x] Represents shape of module/library [correctly](https://www.typescriptlang.org/docs/handbook/declaration-files/library-structures.html)
- [x] `tslint.json` [should contain](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#linter-tslintjson) `{ "extends": "dtslint/dt.json" }`, and no additional rules.
- [x] `tsconfig.json` [should have](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#tsconfigjson) `noImplicitAny`, `noImplicitThis`, `strictNullChecks`, and `strictFunctionTypes` set to `true`.

If changing an existing definition:
- [ ] Provide a URL to documentation or source code which provides context for the suggested changes: <<url here>>
- [ ] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the header.

If removing a declaration:
- [ ] If a package was never on Definitely Typed, you don't need to do anything. (If you wrote a package and provided types, you don't need to register it with us.)
- [ ] Delete the package's directory.
- [ ] Add it to `notNeededPackages.json`.

### CODE REVIEW

File: types/jmuxer/index.d.ts [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+        audio?: Uint8Array;
+        video?: Uint8Array;
+        duration?: number;
+    }
+}
+
+declare class JMuxer {
+    constructor(options: JMuxer.Options);
+    feed(data: JMuxer.Feeder): void;
+    createStream(): any;
```
Comment (Lillifee) [inline types/jmuxer/index.d.ts:27]: Is there a possibility to type the return value of createStream to `Duplex` or `Writeable` without a referencing node?

I searched some other DefinitelyTyped packages and found that:
```
/// <reference types="node" />
import { Duplex } from "stream";
```

Unfortunately, this approach doesn't work in my project.
I wouldn't spend too much time and keep the any.

Comment (samirkumardas) [reply types/jmuxer/index.d.ts:30]: @Lillifee Thank you for your feedback. I tried Duplex earlier as well but did not work. But the comment `/// <reference types="node" />` did the trick. It is working fine now. You need to review it again for this new change.

Comment (Lillifee) [reply types/jmuxer/index.d.ts:30]: Really nice, but you accidentally added the Duplex to the ```destroy``` instead of the ```createStream``` function.

Comment (samirkumardas) [reply types/jmuxer/index.d.ts]: oops! I have corrected it. Thank you for your patience.

**Review by Lillifee — APPROVED** [review]

**Pull request merged**